apps/files/lib/Capabilities.php

```diff
@@ -10,18 +10,20 @@
 use OC\Files\FilenameValidator;
 use OCA\Files\Service\ChunkedUploadConfig;
 use OCP\Capabilities\ICapability;
+use OCP\Files\Conversion\IConversionManager;
 
 class Capabilities implements ICapability {
 
 	public function __construct(
 		protected FilenameValidator $filenameValidator,
+		protected IConversionManager $fileConversionManager,
 	) {
 	}
 
 	/**
 	 * Return this classes capabilities
 	 *
-	 * @return array{files: array{'$comment': ?string, bigfilechunking: bool, blacklisted_files: list<mixed>, forbidden_filenames: list<string>, forbidden_filename_basenames: list<string>, forbidden_filename_characters: list<string>, forbidden_filename_extensions: list<string>, chunked_upload: array{max_size: int, max_parallel_count: int}}}
+	 * @return array{files: array{'$comment': ?string, bigfilechunking: bool, blacklisted_files: list<mixed>, forbidden_filenames: list<string>, forbidden_filename_basenames: list<string>, forbidden_filename_characters: list<string>, forbidden_filename_extensions: list<string>, chunked_upload: array{max_size: int, max_parallel_count: int}, conversions: array<string, mixed>}}
```

Comment:
```suggestion
	 * @return array{files: array{'$comment': ?string, bigfilechunking: bool, blacklisted_files: list<mixed>, forbidden_filenames: list<string>, forbidden_filename_basenames: list<string>, forbidden_filename_characters: list<string>, forbidden_filename_extensions: list<string>, chunked_upload: array{max_size: int, max_parallel_count: int}, file_conversions: list<array{from: string, to: array<string, string>}>}}
```
I believe this is the correct typing.